### DISCUSSION TITLE
Regarding Zip submission

### DISCUSSION THREAD

**Discussion by DelusionArun** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#17] [Q&A] [answered]
Should the github-url.txt should be outside of the project folder? 
And do we need to add TAs as collaborators after deadline is over, or will the usernames be given this week?

If we made slight changes to puzzles for testing, do we need to revert it back to original structure before zipping the project file?

**Comment by Derppening** [comment, marked as answer]
> Should the github-url.txt should be outside of the project folder?

Put it at the *root* of the project folder, i.e. at the top-level of the project folder.

> And do we need to add TAs as collaborators after deadline is over, or will the usernames be given this week?

I will make an announcement before Friday about this arrangement.

> If we made slight changes to puzzles for testing, do we need to revert it back to original structure before zipping the project file?

You do not need to revert the puzzles if you have changed them.

**Reply by DelusionArun** [reply]
So, it will be somewhere below the gitignore? 
Do we have any particular requirement regarding the naming of project zip, or is any name fine for the zip?

**Reply by Derppening** [reply]
> So, it will be somewhere below the gitignore?

In the same folder/directory as `.gitignore` will be fine.

> Do we have any particular requirement regarding the naming of project zip, or is any name fine for the zip?

Any name will be fine.

**Reply by DelusionArun** [reply]
Thank you for clarifying
